mlflow/store/tracking/abstract_store.py

```diff
@@ -329,6 +329,22 @@ def get_trace_info(self, trace_id: str) -> TraceInfo:
         """
         raise NotImplementedError
 
+    def get_trace(self, trace_id: str, allow_partial: bool = False) -> Trace:
```

Comment:
```suggestion
    def get_trace(self, trace_id: str, *, allow_partial: bool = False) -> Trace:
```

to prevent code like `get_trace(..., True)`